src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -314,12 +311,19 @@ class CoreWorkerDirectTaskSubmitter {
     // Check whether the pipelines to the active workers associated with a
     // SchedulingKeyEntry are all full.
     inline bool AllPipelinesToWorkersFull(uint32_t max_tasks_in_flight_per_worker) const {
-      return total_tasks_in_flight ==
+      return total_tasks_in_flight >=
              (active_workers.size() * max_tasks_in_flight_per_worker);
     }
 
     // Check whether there exists at least one task that can be stolen
```

Comment:
Sorry forgot to mention in the previous review, could you add a comment here to say that this will always return false when `max_task_in_flight == 1`?